scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -69,6 +74,9 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
     :param logger:                                  Optional logger to use to log the process and errors
     """
 
+    is_installer_sdk = manifest.is_sdk_engine(engine_path=ctx.engine_path)
+    project_name = manifest.get_project_json_data(project_path=ctx.project_path)['project_name']
```

Comment:
We should check the project.json file was able to be found before querying the project_name key.

That way if the project path isn't pointing to a valid project with a project.json, we can log an error message when the `logger` is initialized below.

```suggestion
    project_name = None
    project_json_data = manifest.get_project_json_data(project_path=ctx.project_path)
    if project_json_data:
        project_name = project_json_data.get('project_name', None);
```

Reply:
actually I might strip out this logic. Upon further review, I realized we already store the project name in the context before the export function is called. I also realized I forgot to include the change that generates the project json stub. That's coming in the next push.